src/lib/stitching/gravity/schema.ts

```diff
@@ -54,13 +44,11 @@ export const executableGravitySchema = () => {
     "User",
     "UsersConnection",
     "UsersConnectionEdge",
+    "MarketingCollection",
```

Comment:
Oh! We should clean things up in Gravity as well (unfortunately we don't have the ability to check if a particular field is being requested in DataDog AFAIK)

Reply:
that makes sense!